presto-spi/src/main/java/io/prestosql/spi/security/Identity.java

```diff
@@ -25,30 +27,30 @@
 
 public class Identity
 {
-    private final String user;
+    private final Name user;
```

Comment:
Let's do not change this in the first step. I would fist introduce unused `Name` (or `Identifier`) to SPI. Then I would make parser to support `Names` without modifying SPI, then slowly modify part by part of SPI along with engine incrementally considering backward compatibility (https://github.com/prestosql/presto/issues/17#issuecomment-469402101) in each step.
If you do a lot of this in one step then it is easy to overlook some bits.